packages/lib-classifier/src/components/Classifier/components/Banners/Banners.js

```diff
@@ -11,14 +11,32 @@ function useStores(stores) {
   const { classifierStore } = stores ?? React.useContext(MobXProviderContext)
   return {
     subject: classifierStore.subjects.active,
+    subjects: classifierStore.subjects,
     workflow: classifierStore.workflows.active
   }
 }
 
+const environment = process.env.APP_ENV
```

Comment:
This is intentional. `APP_ENV` is the Docker build/deploy environment. We want the Next and Previous buttons visible on staging, for Sam to test, but not live in production. Both our staging and production environments use production webpack builds (`NODE_ENV = production`) so I've used `APP_ENV` to discriminate between them.

Reply:
We have documented:

> NODE_ENV: the webpack build mode for libraries and the NextJS apps (production, development or undefined.)
> APP_ENV: sets the Sentry environment (development, staging or production) when reporting errors.

If this is inaccurate, then it needs to be updated.

Reply:
🤷🏻‍♂️ `APP_ENV` is the deployment environment. I think the conversation is wandering outside the scope of this PR.

Reply:
I double checked this in the Dockerfile, the Jenkinsfile and the kubernetes templates. The Readme is wrong. `APP_ENV` is the deployment environment (production, staging or development.) It gets logged with Sentry errors so that we can tell whether an error originates in production, staging or development. Production and staging both use production builds, and a local development server might also be running a production build in order to debug it.